components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -742,33 +745,47 @@ private String handleOAuthAuthorizationRequest(String clientId, HttpServletReque
 
         boolean forceAuthenticate = false;
         boolean checkAuthentication = false;
-
         // values {none, login, consent, select_profile}
+        String[] arrPrompt = new String[]{"none", "login", "consent", "select_profile"};
```

Comment:
Use a constant for `select_account` and use the prompt constants for array initialization.